### ISSUE TITLE
Minimum size doesn't fit 320 resolution device

### ISSUE THREAD

**Issue by umbobabo** [the-economist-editorial/component-scenechanger#1]
Tested on iPhone4  - Safari

The widget doesn't adapt itself to the small resolution.
Anyway, for this small resolution, this kind of widget shouldn't probably have the left padding but occupy all the available horizontal space.
This solution should probably applied at layout level, anyway the widget should be fully responsive.

![screen shot 2015-08-03 at 12 38 47](https://cloud.githubusercontent.com/assets/752680/9036862/8159054a-39dd-11e5-8fc0-34ae00ecd0e0.png)

**Comment by Economist-Editorial** [comment]
This is fixed on a branch I am working on that refactors the gallery and
the screens hanger (now renamed Pager). I will merge this tonight.

On Monday, 3 August 2015, Umberto <notifications@github.com> wrote:

> Tested on iPhone4 - Safari
>
> The widget doesn't adapt itself to the small resolution.
> Anyway, for this small resolution, this kind of widget shouldn't probably
> have the left padding but occupy all the available horizontal space.
> This solution should probably applied at layout level, anyway the widget
> should be fully responsive.
>
> [image: screen shot 2015-08-03 at 12 38 47]
> <https://cloud.githubusercontent.com/assets/752680/9036862/8159054a-39dd-11e5-8fc0-34ae00ecd0e0.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/the-economist-editorial/component-scenechanger/issues/1>
> .
>


-- 
*Angelo Repole*
*Head of Front End development*

The Economist - www.economist.com
+44 20 7830 7073
@angelorep

-- 
*This e-mail may contain confidential material. If you are not an intended 
recipient, please notify the sender and delete all copies. It may also 
contain personal views which are not the views of The Economist Group. We 
may monitor e-mail to and from our network.*

*Sent by a member of The Economist Group. The Group's parent company is The 
Economist Newspaper Limited, registered in England with company number 
236383 and registered office at 25 St James's Street, London, SW1A 1HG. For 
Group company registration details go to http://legal.economistgroup.com 
<http://legal.economistgroup.com> *

**Comment by angelorepole** [comment]
This is fixed on a branch I am working on that refactors the gallery and
the screens hanger (now renamed Pager). I will merge this tonight.
…

**Comment by dhounam** [comment]
Yes, issue is with the Gallery's css, which sets a minimum width of 300px
